DEVELOPING_TUTORIALS.md

```diff
@@ -602,6 +612,73 @@ The properties exported from this file depend on the lesson type, as follows:
   - `question`: String - mandatory - _the question to be answered by the user_ ([detailed docs](#create-multiple-choice-quizzes-in-your-javascript-file-skip-for-coding-exercises-and-text-only-lessons))
   - `choices`: Array - mandatory - _the potential answers between which the user must choose_ ([detailed docs](#create-multiple-choice-quizzes-in-your-javascript-file-skip-for-coding-exercises-and-text-only-lessons))
 
+### `utils` module
+
+This module is a set of utils designed to be re-used accross the validation code of different tutorials.
+
+- `utils.format`
```

Comment:
Switch these to headers instead of bullets so you can link to them from relevant parts of instructions. Reference from other spots in the instructions where the specific util could be helpful.